addons/mail/models/mail_thread.py

```diff
@@ -562,6 +566,15 @@ def _track_set_log_message(self, message):
         for id_ in self.ids:
             body_values[id_] = message
 
+    def _track_get_message(self, tracked_fields):
```

Comment:
``_track_get_default_log_message`` ? otherwise it is a bit hard to distinguish _track_set_log_message and _track_get_message maybe ?